web/docker/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #-----------------------------    BUILD STAGE   ------------------------------#
 ###############################################################################
 
-FROM python:2.7.16-slim-stretch as builder
+FROM python:3.7-slim-stretch as builder
```

Comment:
What do you think about using 3.6 as a base image instead of 3.7, that is the minimal required version.

Reply:
Okay, I changed it to 3.6.